include/psa/crypto.h

```diff
@@ -834,7 +834,6 @@ psa_status_t psa_copy_key(psa_key_handle_t source_handle,
  * \retval #PSA_ERROR_COMMUNICATION_FAILURE
  * \retval #PSA_ERROR_HARDWARE_FAILURE
  * \retval #PSA_ERROR_CORRUPTION_DETECTED
- * \retval #PSA_ERROR_STORAGE_FAILURE
```

Comment:
You've removed `STORAGE_FAILURE` from too many functions.

`psa_copy_key` must access storage if either the source or destination key is persistent.

verify/finish functions for MAC, cipher and AEAD use a key and therefore can access storage in implementations where key material is not systematically kept in RAM during multipart operations.

Reply:
Sorry I'm confused. According to this PR and my copy of the `STORAGE_FAILURE` failure hasn't been removed at all from psa_copy_key, or any of the mac verify/finish functions. I can see that Github has a little Outdated label on your comment above. I might be mistaken but perhaps you're looking at a stale snapshot?

Reply:
Oh, my bad. When I posted this, I was just looking at the next-to-last commit via the git diff, and git diffs are misleading with documentation above the declaration: git infers the name of the function containing a diff fragment but that's the last declared function, not the function that's being documented. So the function _after_ `psa_copy_key` had `STORAGE_FAILURE` removed, etc., which is not a problem. Sorry about that.

Reply:
No problem, I've also found it misleading when reviewing diffs.